### PR TITLE
remove tracking in order to avoid having to show gdpr consent banner

### PR DESCRIPTION
**Pull request by taniki**

### CODE REVIEW

**Comment by abulte** [comment]
ping @XavierJp

**Comment by XavierJp** [comment]
Hi ! what is the expected impact on the API carrousel feature ?

And I am curious: was the tracking using matomo ? Does matomo now require a consent banner ?

**Comment by taniki** [comment]
yeah tracking was using matomo and should be remove if we want to keep a banner free experience since it is not considered as "strictement nécessaire".

**Comment by taniki** [comment]
as a matter of facting. this is more of a tracking issue than a matomo one :)

**Comment by taniki** [comment]
no impact on the api feature except removing usage statistics.

**Comment by XavierJp** [comment]
Great thanks for the explanations

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Current (in progress)
 
-- Nothing yet
+- Remove tracking in order to avoid having to show gdpr consent banner [#517](https://github.com/etalab/udata-gouvfr/pull/517)
```
Comment (abulte) [inline CHANGELOG.md:5]:
```suggestion
- Keep tracking to strictly necessary [#517](https://github.com/etalab/udata-gouvfr/pull/517)
```

**Review by abulte — APPROVED** [review]

**Pull request merged**